### PR TITLE
added info in readme around ESM

### PR DESCRIPTION
**Pull request by bizob2828**
## Proposed Release Notes
 * Added information on using ESM loader to README.md

## Links
Closes NEWRELIC-3118

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,6 +62,28 @@ If you cannot control how your program is run, you can load the `newrelic` modul
...
+
+### Setup
+
+ 1. If you rely on a configuration file to run the agent, you must rename the file from `newrelic.js` to `newrelic.cjs` so it can be properly loaded.  All the contents of the configuration file will behave the same once you rename. See [CommonJS modules in ESM](https://nodejs.org/api/modules.html#enabling) for more details. 
+
+```sh
+$ mv newrelic.js newrelic.cjs
+```
+
+ 2. To run your program pass the newrelic ESM loader with the `--experimental-loader` flag.
```
Comment (mrickard) [inline README.md:79]: While this is accurate, it seems a bit terse. How about this?

> 2. To use the newrelic ESM loader, start Node with the `--experimental-loader` flag and a path to the loader file, like this:

File: README.md [hunk 2]
```diff
@@ -58,6 +62,28 @@ If you cannot control how your program is run, you can load the `newrelic` modul
     /* ... the rest of your program ... */
 ```
 
+## ES Modules
```
Comment (jordigh) [inline README.md:65]: I'd expand the acronym, ECMAScript Modules.

File: README.md [hunk 3]
```diff
@@ -58,6 +62,28 @@ If you cannot control how your program is run, you can load the `newrelic` modul
...
+
+If your application is written with `import` and `export` statements and is not transpiled to javascript, you are using [ES Modules](https://nodejs.org/api/esm.html#modules-ecmascript-modules) and must bootstrap the agent in a different way.
```
Comment (jordigh) [inline README.md:67]: "and is not transpiled" is a bit of a funny way of saying "and it's not typescript", as if import and export weren't javascript too.

How about, "if your application is using import and export modules in javascript"? Or should we explicitly say "and it's not typescript"? Not sure what other things could use import and export.

File: README.md [hunk 4]
```diff
@@ -79,6 +105,8 @@ There are several modules included within the Node.js agent to add more instrume
  * [@newrelic/superagent](https://github.com/newrelic/node-newrelic-superagent): Provides instrumentation for [superagent](https://github.com/visionmedia/superagent) npm package.
  * [@newrelic/native-metrics](https://github.com/newrelic/node-native-metrics): Provides hooks into the native v8 layer of Node.js to provide metrics to the Node.js agent.
 
+ 
+
```
Comment (jordigh) [inline README.md:109]: Stray whitespace change?

File: README.md (hunk 3 above)
Comment (bizob2828) [reply README.md:67]: you can be using babel to transpile native javascript, not sure why you would sitll want to but you can. but i'll just be more concise like "if your application is using import and export modules in javascript"

File: README.md (hunk 2 above)
Comment (bizob2828) [reply README.md:65]: blah. ok just this once 😄

**Review by coreyarnold — APPROVED** [review]
changes to the readme look good and are pretty clear to me. 👍

**Pull request merged**